sklearn/metrics/_plot/confusion_matrix.py

```diff
@@ -70,6 +71,7 @@ def plot(self, *, include_values=True, cmap='viridis',
             Axes object to plot on. If `None`, a new figure and axes is
             created.
 
+        colorbar : matplotlib colorbar, default=True
```

Comment:
You are adding a new parameter, a useful resource is the [scikit-learn contributing documentation](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation), in order to standardize the parameter description.
Also, could you please add a `versionadded` directive? You can check #15426 for examples about how this should be addressed.